src/Microsoft.PowerShell.Commands.Diagnostics/GetEventCommand.cs

```diff
@@ -361,6 +362,13 @@ public SwitchParameter Oldest
         private const string propClose = "]";
         private const string filePrefix = "file://";
         private const string NamedDataTemplate = "((EventData[Data[@Name='{0}']='{1}']) or (UserData/*/{0}='{1}'))";
+        private const string DataTemplate = "(EventData/Data='{0}')";
+        private const string SystemTimeTemplate2 = "(System/TimeCreated[@SystemTime&gt;='{0}' and @SystemTime&lt;='{1}'])";
+        private const string SystemTimeTemplate = "(System/TimeCreated[@SystemTime&lt;='{0}'])";
```

Comment:
You will need another const variable to hold `"(System/TimeCreated[@SystemTime&gt;='{0}'])"`. Maybe the variables names can be more descriptive. Instead of `*2` and `*3`, how about `SystemTimeStartTemplate`, `SystemTimeEndTemplate`, and `SystemTimePeriodTemplate`?